aspnetcore/signalr/dotnet-client/sample/SignalRChatClient/packages.config

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
I kinda want to clean this up by showing PackageReference but that isn't the default so I don't know what's better. See https://github.com/aspnet/SignalR-samples/blob/7a0dafb7f58dea036ba1cbdf252edf42692909bb/WindowsFormsSample/WindowsFormsSample/WindowsFormsSample.csproj#L83

Reply:
What about staying with defaults for ease of readership and if folks don't like it, then we can change?